lib/resolvers/post_query/commentsByPost.js

```diff
@@ -2,18 +2,53 @@ const { NotFoundError } = require('errors');
 const requestContext = require('talawa-request-context');
 
 const Comment = require('../../models/Comment');
+const Organization = require('../../models/Organization')
 
 module.exports = async (parent, args) => {
   const commentFound = await Comment.find({ post: args.id })
     .populate('creator', '-password')
     .populate('post')
     .populate('likedBy');
-  if (!commentFound) {
+  //comment does not exist
+  if (!commentFound.length) {
     throw new NotFoundError(
-      requestContext.translate('comment.notFound'),
+      process.env.NODE_ENV !== 'production'
```

Comment:
change the check into the ones located in the constants file (if any, isn't present then add)